packages/tradershub/src/features/cfd/screens/CFDCompareAccounts/CompareAccountsPlatformLabel.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { qtMerge, Text } from '@deriv/quill-design';
+import { TPlatforms } from '../../../../types';
+import {
+    CompareAccountsPlatformLabelClass,
+    CompareAccountsPlatformLabelTextColorClass,
+    TCompareAccountsPlatformLabelClassProps,
+    TCompareAccountsPlatformLabelTextClassProps,
+} from './CompareAccounts.classnames';
+import { getPlatformType } from './CompareAccountsConfig';
+import { platformLabel } from './constants';
+
+type TCompareAccountsPlatformLabel = {
+    platform: TPlatforms.All;
+};
+
+const CompareAccountsPlatformLabel = ({ platform }: TCompareAccountsPlatformLabel) => {
+    const platformType = getPlatformType(platform);
+
+    return (
+        <div
+            className={qtMerge(
+                CompareAccountsPlatformLabelClass({
+                    background: platformType,
```

Comment:
rename the objects to have a basic understanding of what we want to achieve here from plaformType.